packaging/test_irods_client_rest_cpp.py

```diff
@@ -68,6 +68,58 @@ def test_access_with_default_arguments(self):
             finally:
                 os.remove(file_name)
                 admin.assert_icommand(['irm', '-f', file_name])
+    def test_logical_path_rename(self):
+        base_path = '/tempZone/home/rods'
```

Comment:
replace with something like `admin.home_collection`

Reply:
Seems like the rest of the tests take the approach of creating things within the pwd, and getting that in the form of `execute_command`